src/utils/uniqueBySlug.util.js

```diff
@@ -0,0 +1,15 @@
+export default (arr = []) => arr.sort((a, b) => {
+  if (a.slug < b.slug) {
+    return -1;
+  }
+  if (b.slug < a.slug) {
+    return 1;
+  }
+  return 0;
+}).reduce((acc, curr) => { // remove duplicates
```

Comment:
Is there a case where you might have duplicate slugs?

Reply:
This is explained in the comment above @lukorito, please go through the implementation of the reducer and you'll see where the `banner` articles can be duplicated.